lib/hooks/useEvent.lua

```diff
@@ -27,47 +95,44 @@ end
 		end
 	end
 	```
-
 	Returns an iterator function that returns an ever-increasing number, starting at 1, followed by any event arguments
 	from the specified event.
-
 	Events are returned in the order that they were fired.
-
 	:::caution
 	`useEvent` keys storage uniquely identified by **the script and line number** `useEvent` was called from, and the
 	first parameter (instance). If the second parameter, `event`, is not equal to the event passed in for this unique
 	storage last frame, the old event is disconnected from and the new one is connected in its place.
-
 	Tl;dr: on a given line, you should hard-code a single event to connect to. Do not dynamically change the event with
 	a variable. Dynamically changing the first parameter (instance) is fine.
-
 	```lua
 	for _, instance in pairs(someTable) do
 		for i, arg1, arg2 in useEvent(instance, "Touched") do -- This is ok
-
 		end
 	end
-
 	for _, instance in pairs(someTable) do
 		local event = getEventSomehow()
 		for i, arg1, arg2 in useEvent(instance, event) do -- PANIC! This is NOT OK
-
 		end
 	end
 	```
 	:::
-
 	If `useEvent` ceases to be called on the same line with the same instance and event, the event connection is
 	disconnected from automatically.
-
 	You can also pass the actual event object instead of its name as the second parameter:
-
 	```lua
 	useEvent(instance, instance.Touched)
-
 	useEvent(instance, instance:GetPropertyChangedSignal("Name"))
 	```
-
+	Additionally, `useEvent` supports custom events as well (through duck typing), so you can pass in an object with a
```

Comment:
We don't need to specify this, kind of irrelevant for the users of the library.

Reply:
You probably don't need to specify that the function is found through duck typing, but it might actually be useful to know that the only supported methods are "Connect", "connect", and "on"